main.py

```diff
@@ -0,0 +1,25 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from webdriver_manager.chrome import ChromeDriverManager
+
+browser = webdriver.Chrome(ChromeDriverManager().install())
+browser.implicitly_wait(30)
+browser.get('https://www.twitch.tv/')
+
+browser.find_element_by_css_selector('[data-a-target="login-button"]').click()
+body = browser.find_element_by_tag_name('body')
+modalClass = 'ReactModal__Body--open'
+
+assert modalClass in body.get_attribute('class')
```

Comment:
in e2e tests, try to avoid assertions based on class values

Reply:
@byteofmydream Understood, thank you for a review.